tcrypto/types/src/ecdsa_p256.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Elliptic curve digital signature algorithm (ECDSA) types for curve P-256
+
+use mc_sgx_core_types::new_type_accessors_impls;
+use mc_sgx_tcrypto_sys_types::{sgx_ec256_public_t, SGX_ECP256_KEY_SIZE};
+
+/// Elliptic curve digital signature algorithm (ECDSA) curve P-256 public key
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+#[repr(transparent)]
+pub struct PublicKey(sgx_ec256_public_t);
```

Comment:
This type in particular is needed by tservice so this is why (some of) tcrypto is coming in now